polaris/loader/load.py

```diff
@@ -35,8 +35,9 @@ def load_dataset(path: str, verify_checksum: ChecksumStrategy = "verify_unless_z
 
     if not is_file:
         # Load from the Hub
-        client = PolarisHubClient()
-        return client.get_dataset(*path.split("/"), verify_checksum=verify_checksum)
+        with PolarisHubClient() as client:
+            client.ensure_active_token()
```

Comment:
**Comment**: Needed, since we never call `polaris login` when signing in with `POLARIS_USER` and `POLARIS_PASSWORD`.